PyInstaller/hooks/hook-eyed3.plugins.py

```diff
@@ -0,0 +1,12 @@
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules
+from os.path import basename, splitext
+
+plugins = filter(lambda data: basename(data[0])[0] not in ('_', '.') and data[0].endswith('.py'),
+        collect_data_files('eyed3.plugins', include_py_files=True))
```

Comment:
Why not using  `collect_submodules`, which already collect submodules only?

Reply:
I used the same process that the eyed3 collects its plugins.
eyed3 just collects '.py' files without initial '_' or '.'. `collect_submodule` may collect some useless modules that just has '.pyc' or '.pyo' but no '.py'.